deb/build_deb.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+#
+# Script for deb packaging dvbtee
+#
+# Tested on Ubuntu 14.04
+#
+# Author: Pasha Mesh <pasha.mesh@gmail.com>
+#
+
+PATH='/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games'
+
+current_dir="$(dirname "$(readlink -f "$0")")"
+compiled_base_dir="$(readlink -f "${current_dir}/../")"
+
+pkg_install_dir='/opt/dvbtee'
+
+err() {
+  echo -e "[$(date +'%Y-%m-%dT%H:%M:%S%z')]: \e[32m${@}\e[0m" >&2
+}
+
+main() {
+  # Run sources build
+  cd "${compiled_base_dir}" && ./build.sh
```

Comment:
let's move away from build.sh ... This script builds a local copy of libdvbpsi, but if we're packaging libdvbtee as a deb, then we should also build it against the deb version of libdvbpsi

instead of ./build.sh, lets do something along the lines of:
"autoreconf -i && ./configure && make"